sphinx/tutorial/rose/suites.rst

```diff
@@ -402,7 +267,7 @@ See the :ref:`Cheat Sheet` for more information.
 
       .. code-block:: diff
 
-          [cylc]
+          [scheduler]
               UTC mode = True
               [[parameters]]
```

Comment:
This has changed to `flow.cylc[task parameters]`

Reply:
Yes.